doc/source/development/building_from_source.rst

```diff
@@ -1039,6 +1039,13 @@ It can be detected with pkg-config.
 
     Control whether to use LibKML. Defaults to ON when LibKML is found.
 
+.. option:: LIBKML_MINIZIP_LIBRARY
+
+    Path to a shared or static library file for ``minizip``
+
+.. option:: LIBKML_URIPARSER_LIBRARY
+
+    Path to a shared or static library file for ``urlparser``
```

Comment:
Which one, "uriparser" or "urlparser", is correct?

Reply:
Yep! uriparser is correct - it's very old library.

Also you can look in here:
https://github.com/OSGeo/gdal/blob/master/cmake/modules/packages/FindLibKML.cmake#L70

Reply:
Would it be better to call it as uriparser also in the comment, even it is hard to see that current "library file for ``urlparser``" could make any issues?

Reply:
Oh... you are ritght! my mistake